eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -387,12 +393,31 @@ function GetExistingTags($apiUrl) {
   }
 }
 
-# Walk across all build artifacts, check them against the appropriate repository, return a list of tags/releases
-function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha,  $continueOnError = $false) {
-  $pkgList = [array]@()
+# Retrieve release tag for artiface package. If multiple packages, then output the first one.
+function RetrieveReleaseTag($pkgRepository, $artifactLocation, $continueOnError = $true) {
+  if (!$artifactLocation) {
+    Write-Warning "Please provide the artifact location which to retrieve the releaes tag."
```

Comment:
I wouldn't output the warning here just return empty like the other places. We might want to add a warning at the calling side whenever the release tag is empty.

Reply:
Sure. Will remove the comments.